### DISCUSSION TITLE
Welcome to Ejenda Discussions!

### DISCUSSION THREAD

**Discussion by GrahamSH-LLK** [Ejenda/Ejenda#61] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheSeagullDev** [comment]
Hi, I'm Thingyman!
I use Ejenda every day and help implement feedback and new ideas into Ejenda as well as help manage our development flow! If you have any ideas or suggestions, don't hesitate to reach out to me!
